### ISSUE TITLE
In eml-dataTable caseSensitive and orientation are modeled differently

### ISSUE THREAD

**Issue by mbjones** [NCEAS/eml#84]
---

Author Name: **David Blankman** (David Blankman)
Original Redmine Issue: 597, https://projects.ecoinformatics.org/ecoinfo/issues/597
Original Date: 2002-09-19
Original Assignee: David Blankman

---

In eml-dataTable caseSensitive and orientation are modeled differently. 

Orientation has an attribute of columnOrRow with  an enumeration of columnMajor,
rowMajor.

caseSensitive has no attribute, just an enumeration of yes,no.

Is there a purpose behind the extra complexity of orientation?

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **David Blankman** (David Blankman)
Original Date: 2002-10-02T00:00:26Z

---

Deleted the orientation attribute. Made a restriction in the element of
"columnMajor" or "rowMajor". Updated the documentation to reflect the change. In
the documentation/description I gave an example of the same data represented in
a columnMajor table and the same data in a rowMajor table.
